packages/hash/api/src/graphql/resolvers/entity/linkGroups.ts

```diff
@@ -6,15 +6,15 @@ import { Entity, Link } from "../../../model";
 import { DbClient } from "../../../db";
 import { DbUnknownEntity } from "../../../db/adapter";
 
-export const DEFAULT_LINK_DEPTH = 2;
+export const DEFAULT_LINK_DEPTH = 1;
```

Comment:
the `blockGraph` message requires a `depth` parameter. 

I originally made this consistent because I had in my head that `depth` referred to both `linkedEntities` and `linkedAggregations`, but actually `linkedAggregations` doesn't even have a depth parameter. It should, so I've added [a task](https://app.asana.com/0/1201095311341924/1202566735536511/f) to add it.

I'm just going to leave this at 1 so it's consistent with `linkedAggregations` as I expect it'd be easier to reason about.